cupy/__init__.py

```diff
@@ -1,3 +1,4 @@
+from cupy.lib import emath  # NOQA
```

Comment:
Does moving this line just after `from cupy import testing` below cause any problem?